test/src/main/java/org/apache/accumulo/test/functional/KerberosRenewalIT.java

```diff
@@ -200,7 +199,21 @@ private void createReadWriteDrop(AccumuloClient client) throws TableNotFoundExce
           new Key("a", "b", "c").compareTo(entry.getKey(), PartialKey.ROW_COLFAM_COLQUAL),
           "Did not find the expected key");
       assertEquals("d", entry.getValue().toString());
-      client.tableOperations().delete(table);
     }
+    client.tableOperations().delete(table);
```

Comment:
this could be in a finally block, but likely not necessary since this is a test

Reply:
Do you think that would be an improvement? It can be easily added if so.

Reply:
I don't know that it would be an improvement as you are getting a unique table name at the start of the test. If this test fails, it's unlikely that the table existing would cause an issue with another test or this test being repeated. If this were not test code, I would say putting it in a finally block would be needed.